name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/DkbPDFExtractor.java

```diff
@@ -164,9 +164,18 @@ private void addBuySellTransaction()
                 .assign((t, v) -> {
                     // Workaround for bonds
                     if (v.get("notation") != null && !v.get("notation").equalsIgnoreCase("Stück"))
-                        t.setShares((asShares(v.get("shares")) / 100));
+                    {
+                        // TODO: https://github.com/buchen/portfolio/issues/2684
+                        BigDecimal shares = asBigDecimal(v.get("shares"));
+                        shares = shares.divide(BigDecimal.valueOf(100));
+                        v.put("shares", shares.toString().replace(".", ","));
+
+                        t.setShares(asShares(v.get("shares")));
+                    }
```

Comment:
Ich habe den Teil etwas vereinfacht. Wenn wir schon die Zahl haben, dann brauchen wir die nicht wieder in einen String umzuwandeln.

```
BigDecimal shares = asBigDecimal(v.get("shares"));
t.setShares(Values.Share.factorize(shares.doubleValue() / 100));
```